CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# CONTRIBUTING
+
+We welcome contributions from everyone!
+
+## Getting Started
+
+Make sure you have NodeJS 0.10 or higher and npm installed.
```

Comment:
This is pretty nitpicky, but if this is the case we should modify the Travis build runner to run on node 0.10.  I'd also say that node 0.10 is super old and it would be fine to update this to node 4.

Reply:
I'll update this doc and open and see if this is still an issue with the generator